hedera-node/src/test/java/com/hedera/services/utils/SignedTxnAccessorTest.java

```diff
@@ -146,4 +152,14 @@ void throwsOnUnsupportedCallToGetScheduleRef() {
 		// expect:
 		Assertions.assertThrows(UnsupportedOperationException.class, subject::getScheduleRef);
 	}
+
+//	@Test
```

Comment:
Not sure what this repo's stance is on commented out tests, but if this test does compile slapping a `@Disabled` might be better.